native/cocos/bindings/jswrapper/config.cpp

```diff
@@ -23,72 +23,15 @@
  THE SOFTWARE.
 ****************************************************************************/
 
-#include "config.h"
-#include <algorithm>
-#include <cstdio>
-#include "base/memory/Memory.h"
-
-#if defined(_WIN32) && defined(_WINDOWS)
-
-    #include <windows.h>
-
-static void _winLog(FILE *fp, const char *format, va_list args) {
-    static const int MAX_LOG_LENGTH = 16 * 1024;
-    int bufferSize = MAX_LOG_LENGTH;
-    char *buf = nullptr;
-
-    do {
-        buf = ccnew char[bufferSize];
-        if (buf == nullptr)
-            return; // not enough memory
-
-        int ret = vsnprintf(buf, bufferSize - 3, format, args);
-        if (ret < 0) {
-            bufferSize *= 2;
-
-            delete[] buf;
-        } else
-            break;
-
-    } while (true);
-
-    int pos = 0;
-    auto len = static_cast<int>(strlen(buf));
-    char tempBuf[MAX_LOG_LENGTH + 1] = {0};
-    WCHAR wszBuf[MAX_LOG_LENGTH + 1] = {0};
-
-    do {
-        std::copy(buf + pos, buf + pos + MAX_LOG_LENGTH, tempBuf);
-
-        tempBuf[MAX_LOG_LENGTH] = 0;
-
-        MultiByteToWideChar(CP_UTF8, 0, tempBuf, -1, wszBuf, sizeof(wszBuf));
-        OutputDebugStringW(wszBuf);
-        WideCharToMultiByte(CP_ACP, 0, wszBuf, -1, tempBuf, sizeof(tempBuf), nullptr, FALSE);
-
-        fprintf(fp, "%s", tempBuf);
-        fflush(fp);
-
-        pos += MAX_LOG_LENGTH;
-
-    } while (pos < len);
-
-    fflush(stdout);
-    delete[] buf;
+#include "bindings/jswrapper/config.h"
+#include <cstdarg>
+#include "base/Log.h"
+
+void selogMessage(cc::LogLevel level, const char *tag, const char *format, ...) {
+    char logbuf[512] = {0};
+    va_list argp;
+    va_start(argp, format);
+    (void)std::vsnprintf(logbuf, sizeof(logbuf), format, argp);
```

Comment:
Why add `(void)` prefix before std::vsnprintf ?

Reply:
To ignore return value